### ISSUE TITLE
Spanner datasource need to detect empty string or null.

### ISSUE THREAD

**Issue by katsew** [Mitu217/tamate#88]
Currently, both empty string and null are converted as a string type, but they should be different.

**Comment by castaneai** [comment]
example) sql package

```go
// NullString represents a string that may be null.
// NullString implements the Scanner interface so
// it can be used as a scan destination:
//
//  var s NullString
//  err := db.QueryRow("SELECT name FROM foo WHERE id=?", id).Scan(&s)
//  ...
//  if s.Valid {
//     // use s.String
//  } else {
//     // NULL value
//  }
//
type NullString struct {
	String string
	Valid  bool // Valid is true if String is not NULL
}
```

**Comment by castaneai** [comment]
Closed by #93
